inputs/transmission/miso_tranche2_capacity.ipynb

```diff
@@ -0,0 +1,328 @@
+    "\n",
+    "The plots are on pages 39 (345 kV) and 43 (765 kV) of this [MISO presentation](https://cdn.misoenergy.org/20230308%20PAC%20Item%2007%20Discussion%20of%20765%20kV%20and%20HVDC628088.pdf).\n",
+    "\n",
+    "The plots were converted into datasets using [WebPlotDigitizer](https://apps.automeris.io/wpd4/). The images, WebPlotDigitizer json files, and resulting csv files are all saved in the directory with this notebook."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Load the parameters for typical 345 and 765 kV transmission lines"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 68,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "def load_and_plot_csv(file_path, title):\n",
+    "    \"\"\"\n",
+    "    Loads data from a CSV file, creates a plot of three pairs of x and y values, \n",
+    "    and displays the plot. After plotting to confirm results, the function converts\n",
+    "    distances from miles to meters and saves the loaded CSV data into a DataFrame.\n",
+    "\n",
+    "    Parameters:\n",
+    "    file_path (str): The path to the CSV file to load.\n",
+    "    title (str): The title of the plot.\n",
+    "\n",
+    "    Returns:\n",
+    "    pd.DataFrame: A pandas DataFrame containing the data from the CSV file.\n",
+    "    \"\"\"\n",
+    "    # Load the CSV data into a pandas DataFrame\n",
+    "    data = pd.read_csv(file_path)\n",
+    "   \n",
+    "    # Plot each pair of x and y values with custom markers and colors\n",
+    "    plt.figure(figsize=(10, 6))\n",
+    "    \n",
+    "    # Plot Max, Thermal, and Safe pairs\n",
+    "    plt.plot(data['max-mi'], data['max-MW'], label='Max', marker='o', color='red')\n",
+    "    plt.plot(data['thermal-mi'], data['thermal-MW'], label='Thermal', marker='x', color='blue')\n",
+    "    plt.plot(data['safe-mi'], data['safe-MW'], label='Safe', marker='^', color='green')\n",
+    "    \n",
+    "    # Adding titles and labels\n",
+    "    plt.title(title)\n",
+    "    plt.xlabel('Line Miles')\n",
+    "    plt.ylabel('MVA or MW')\n",
+    "    \n",
+    "    # Show legend\n",
+    "    plt.legend()\n",
+    "    \n",
+    "    # Show the plot\n",
+    "    plt.grid(True)\n",
+    "    plt.show()\n",
+    "\n",
+    "    # Convert the units from miles to meters and rename the columns\n",
+    "    mile_columns = ['max-mi', 'thermal-mi', 'safe-mi']\n",
+    "    for col in mile_columns:\n",
+    "        data[col] = data[col] * 1609.34  # Convert miles to meters\n",
+    "    data = data.rename(columns={         # Rename columns to meters\n",
+    "        'max-mi': 'max-m',\n",
+    "        'thermal-mi': 'thermal-m',\n",
+    "        'safe-mi': 'safe-m',\n",
+    "        'max-MW': 'max-MW',\n",
+    "        'thermal-MW': 'thermal-MW',\n",
+    "        'safe-MW': 'safe-MW'\n",
```

Comment:
```suggestion
```
The `rename` method doesn't require an entry for every column. Since these aren't changed at all, we can safely delete them.

Reply:
> [!NOTE] 
> If you want to adopt this change click the `Add suggestion to batch` button. Once you've added all of the changes you want, you can commit them with the `commit suggestion` button.